### ISSUE TITLE
(Tech Support) Softlock: Future modules can be disabled in factory mode

### ISSUE THREAD

**Issue by willem88836** [willem88836/ktanemodkit#4]
Report copied from discord: 
```
Ok, I do actually have a bug to report now. @willem88836 Tech support can disable modules on other bombs, which is cool, but causes problems in factory mode as it disables modules on future bombs. Running out of time on the needy will fail to re-enable them. So does defusing the current bomb before solving/running the clock on the needy. Solving the needy will re-enable the module as expected

tldr unless the needy is solved every time before the current bomb is defused, it can softlock future bombs

I tested this using the following:
(
  TechSupport
  DoubleOhModule
)

(
  3*skinnyWires
)
```

[Link to Factory](https://steamcommunity.com/sharedfiles/filedetails/?id=1307301431)
